cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/bindings/scripts/ScriptEvaluator.java

```diff
@@ -80,7 +80,13 @@ private Value doEvaluateExpressionJython(String expr,
                                              Set<SystemProperty> systemProperties,
                                              Set<ScriptFunction> functionDependencies) {
         Map<String, Serializable> jythonContext = createJythonContext(context);
-        boolean systemPropertiesDefined = functionDependencies.contains(ScriptFunction.GET_SYSTEM_PROPERTY);
+
+        boolean systemPropertiesDefined = false;
+        if (functionDependencies.contains(ScriptFunction.GET_SYSTEM_PROPERTY) ||
+                functionDependencies.contains(ScriptFunction.GET_SP_VAR)) {
```

Comment:
|| should be on newline not on first line

Reply:
Done